math/min/bitwisemin.go

```diff
@@ -0,0 +1,12 @@
+// bitwisemin.go
+// description: Gives min of two integers
+// details:
+// implementation of finding the minimum of two numbers using only binary operations without using conditions
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see bitwisemin_test.go
+
+package min
+
+func BitwiseMin(a int, b int, base int) int {
```

Comment:
Min is in the name of the package 😄 Just change it to `Bitwise`

```suggestion
func Bitwise(a int, b int, base int) int {
```

I was wondering if you can generalise it to any number of inputs?

Reply:
done